scripts/check_frontend_test_coverage.py

```diff
@@ -63,10 +63,12 @@
     'core/templates/domain/question/question.model.ts',
     'core/templates/expressions/expression-interpolation.service.ts',
     'core/templates/google-analytics.initializer.ts',
+    'core/templates/pages/blog-post-page/blog-post-page.component.ts',
```

Comment:
why are we backtracking on frontend coverage? The Angular migration is a big enough deal that I'm okay with it--just want to make sure we get the reasoning documented